code/modules/reagents/reagent_containers/jug.dm

```diff
@@ -0,0 +1,53 @@
+//Not to be confused with /obj/item/reagent_containers/food/drinks/bottle
+
+
+/obj/item/reagent_containers/glass/jug
+	name = "jug"
+	desc = "A decent sized plastic jug."
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "plastic_jug"
+	item_state = "plastic_jug"
+	amount_per_transfer_from_this = 10
+	possible_transfer_amounts = list(1,2,5,10,20,40,80)
+	container_type = OPENCONTAINER
+	volume = 80
+
+/obj/item/reagent_containers/glass/jug/Initialize()
+	..()
```

Comment:
`. = ..()` here.
Initalize is expected to return a value saying if it has succesfully initalized or if it failed etc